__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -169,3 +169,49 @@ export const Light: StoryComponentType = {
     },
     render: AllFields,
 };
+
+/**
+ * The following story shows what the LabeledField looks like when different
+ * props are set.
+ */
+export const Combinations = (args: PropsFor<typeof LabeledField>) => {
+    const [textFieldValue, setTextFieldValue] = React.useState("");
+
+    return (
+        <View style={{gap: spacing.large_24}}>
+            <LabeledField
+                {...args}
+                label="Label only"
+                field={
+                    <TextField
+                        value={textFieldValue}
+                        onChange={setTextFieldValue}
+                    />
+                }
+            />
+            <LabeledField
+                {...args}
+                label="With description"
+                description="Description"
+                field={
+                    <TextField
+                        value={textFieldValue}
+                        onChange={setTextFieldValue}
+                    />
+                }
+            />
+            <LabeledField
+                {...args}
+                label="With Error"
+                error="Error message"
+                field={
+                    <TextField
+                        value="invalid value"
+                        onChange={() => {}}
+                        validate={() => "Error message"}
+                    />
+                }
+            />
```

Comment:
**thought:** How do you feel about adding a `With Description + Error` example here? I understand that this is covered by previous stories, but maybe it would be good to include it here so folks are aware of all possible combinations when they navigate to this example? (no change required).

Reply:
Good point, I've added it in! 

I will also add more examples to this story so I renamed it to "Scenarios". I'm going to add examples for handling overflowing text when I work on polishing the styling in another PR!